mafiasi/gprot/models.py

```diff
@@ -6,17 +6,38 @@
 
 from mafiasi.teaching.models import Course, Teacher
 
+
 def make_filename(obj, prefix, ext):
     return '{0}/{1}-{2}.{3}'.format(
         prefix, hexlify(os.urandom(16)), obj.pk, ext)
 
+
 def make_gprot_filename(gprot, filename):
     return make_filename(gprot, 'gprot', 'pdf')
 
+
 def make_attachment_filename(attachment, filename):
     ext = attachment.mime_type.split('/')[-1]
     return make_filename(attachment, 'gprot-attachment', ext)
 
+
+class LabelManager(models.Manager):
+
+    def as_choices(self):
+        for label in self.all().order_by('tag'):
+            yield label.pk, label.tag
+
+
+class Label(models.Model):
+    tag = models.CharField(max_length=50)
+    colour = models.CharField(max_length=15, default="default")
```

Comment:
We should check whether we use American or British english in the rest of the code base. I would prefer `color` here, as that is the attribute in CSS etc. as well.